DTS-STN/secure-client-hub
# Replace Header & Footer

## [ADO-140361](https://dev.azure.com/VP-BD/DECD/_workitems/edit/140361)

Fixed [AB#140361](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/140361)

### Changelog - "fix:" for bug fixes, "feat:" for features. Read more about Conventional Commits at https://www.conventionalcommits.org/en/v1.0.0/#summary
feat:Updated app to use local Header implementation
feat:Updated app to use local Footer implementation
feat:Updated app to use local TopNav, Menu, Language switcher, Breadcrumb implementation

### Description of proposed changes:
This PR updates our app to use a local version of the Header (and all the components it encompasses) and Footer instead of pulling from DS. While the original code is still from DS, I've taken out code that doesn't pertain to our project in order to keep things clean. I've also replaced instances of `Image` and `Link` with their corresponding Next.js components.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate through the app and ensure it looks and acts as expected

### Additional Notes
The tests for layout have been skipped in this PR since they are broken at a fundamental level and need to be rewritten. This will be done in a separate PR.
